src/scripts/getFileSha1.js

```diff
@@ -0,0 +1,20 @@
+var sha1 = require('sha1');
```

Comment:
Please change this require path to something like this var sha1 = require('../../node_modules/js-sha1/src/sha1');
SO that the require module can find the the sha1 module based on the relative path. Install the sh1 module in scripts folder should be working but I need all dependencies to be installed in the root